pkg/apiserver/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -259,22 +259,24 @@ func admitDVs(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 		return toAdmissionResponseError(err)
 	}
 
-	client := GetClient()
-	if client != nil {
-		pvcs, err := client.CoreV1().PersistentVolumeClaims(dv.GetNamespace()).List(metav1.ListOptions{})
-		if err != nil {
-			return toAdmissionResponseError(err)
-		}
-		for _, pvc := range pvcs.Items {
-			if pvc.Name == dv.GetName() {
-				klog.Errorf("destination PVC %s/%s already exists", dv.GetNamespace(), dv.GetName())
-				var causes []metav1.StatusCause
-				causes = append(causes, metav1.StatusCause{
-					Type:    metav1.CauseTypeFieldValueDuplicate,
-					Message: fmt.Sprintf("Destination PVC already exists"),
-					Field:   k8sfield.NewPath("DataVolume").Child("Name").String(),
-				})
-				return toRejectedAdmissionResponse(causes)
+	if ar.Request.Operation == v1beta1.Create {
+		client := GetClient()
+		if client != nil {
+			pvcs, err := client.CoreV1().PersistentVolumeClaims(dv.GetNamespace()).List(metav1.ListOptions{})
```

Comment:
nit: rather than retrieve the entire list (which in theory could be lengthy) and iterate over the entire thing; why not just call Get by name?

Reply:
Good point, and I am unable to reproduce the error above while importing, so I think I will update this PR to simply implement the get vs list.